src/6model/reprs/VMArray.c

```diff
@@ -1502,6 +1502,14 @@ void *MVM_VMArray_find_fast_impl_for_jit(MVMThreadContext *tc, MVMSTable *st, MV
                 return vmarray_at_pos_int64;
             }
             break;
+        case MVM_OP_atpos_u:
+            if (kind != MVM_reg_uint64) {
+                return NULL;
+            }
+            if (repr_data->slot_type == MVM_ARRAY_I64) {
```

Comment:
`MVM_ARRAY_I64`?

Reply:
Great catch! Looks like this is something that would just fail silently.